src/pages/cheoljinPark/Main/Aside.js

```diff
@@ -0,0 +1,116 @@
+            <div className="story-title">
+              <div className="story-name">jeong_23</div>
+              <div className="story-time">49분 전</div>
+            </div>
+          </li>
+          <li className="side-story">
+            <div className="story-profile" />
+            <div className="story-title">
+              <div className="story-name">kaede_jp</div>
+              <div className="story-time">24분 전</div>
+            </div>
+          </li>
+          <li className="side-story">
+            <div className="story-profile" />
+            <div className="story-title">
+              <div className="story-name">brian4583</div>
+              <div className="story-time">11분 전</div>
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <div className="side-recommandsbox">
+        <div className="side-recommandstitle">
+          <span className="blur">회원님을 위한 추천</span>{' '}
+          <span>모두 보기</span>
+        </div>
+        <ul className="side-recommandslist">
+          <li className="side-recommand">
+            <div className="recommand-profile" />
+            <div className="recommand-title">
+              <div className="recommand-name">alice</div>
+              <div className="recommand-follower">회원님을 팔로우합니다</div>
+            </div>
+            <button className="recommand-follow">팔로우</button>
+          </li>
+          <li className="side-recommand">
+            <div className="recommand-profile" />
+            <div className="recommand-title">
+              <div className="recommand-name">brand07</div>
+              <div className="recommand-follower">회원님을 팔로우합니다</div>
+            </div>
+            <button className="recommand-follow">팔로우</button>
+          </li>
+          <li className="side-recommand">
+            <div className="recommand-profile" />
+            <div className="recommand-title">
+              <div className="recommand-name">netw0rk23</div>
+              <div className="recommand-follower">회원님을 팔로우합니다</div>
+            </div>
+            <button className="recommand-follow">팔로우</button>
+          </li>
+          <li className="side-recommand">
+            <div className="recommand-profile" />
+            <div className="recommand-title">
+              <div className="recommand-name">hello_9499</div>
+              <div className="recommand-follower">회원님을 팔로우합니다</div>
+            </div>
+            <button className="recommand-follow">팔로우</button>
+          </li>
+          <li className="side-recommand">
+            <div className="recommand-profile" />
+            <div className="recommand-title">
+              <div className="recommand-name">Kellen1</div>
+              <div className="recommand-follower">회원님을 팔로우합니다</div>
+            </div>
+            <button className="recommand-follow">팔로우</button>
+          </li>
+        </ul>
+      </div>
+
+      <div className="side-information">
```

Comment:
```suggestion
      <Footer />
```
Footer 컴포넌트로 분리해주고, div 대신 footer 시멘틱 태그를 사용해보면 좋을 듯!

Reply:
오홍홍 좋아요